packages/services/f-http/src/mockFactory.js

```diff
@@ -15,7 +15,7 @@ export default class StatClient {
      * @param {object} statusCode - The status code to return
      * @param {object} data - The data to return
      */
-    async setupMockResponse (method, url, requestData, statusCode, data) {
+    setupMockResponse (method, url, requestData, statusCode, data) {
```

Comment:
Did you mean to remove `async`?

Reply:
It doesn't look like anything is being `await`ed so this looks okay to me.